### ISSUE TITLE
Improve ourselves 

### ISSUE THREAD

**Issue by TheOnizu** [WebAnimationWorkshops/ama#4]
Hi everybody, thanks so much for opening this , and giving you free time to us i really apreciate it.

I'm Gaoussou, big fan of animation, ( junior frontEnd dev )
I have to ask about how to make my learning curve exponentially increase.
I mean i put my free time looking at how things are build, and look at stuff like :
https://github.com/Jam3/jam3-lesson#other-resources
http://natureofcode.com/
https://github.com/baptistebriel/biggie
https://github.com/facebook/flux
Bottom line i do all of this to be able to be a bad ass creative Developper  as fast as possible and need your point of view on this.
What should i do more and what is your suggestions ?

( sorry for my bad english im french lol :)
Merci !!!

**Comment by sdras** [comment]
Hi @Gaoussou1!

Thanks for posting. I've been asked this before and you might not like the answer. It kind of goes along the lines of "when in doubt, work harder". I think it's really important that you find the work rewarding- I would say "fun" but it's not really as simple as fun, because there are times it's really aggravating. I think rewarding kind of encompasses the pain as well, haha. But the truth is, you have to put the time in. There's a little luck for sure. But doing the work is the biggest thing, sticking with it, following through. And the only way you'll do that in any sustained way is if you find it rewarding.

I'd also say that being good to others is another huge thing. That sounds like mushy kindergarten stuff, but people really do want to work with people they get along with, and you get much more done with some sort of shared understanding for people around you. Take, for instance, one of your examples, "The Nature of Code". That book is by Daniel Shiffman, and you won't find a nicer or more endearing human around. Part of his success, and the reason why people *love* watching him on youtube on Coding Rainbow/Coding Train, is how likeable he is. He's easy to watch because you feel like you're in it together, not that he's telling you how smart he is (he's really really really very smart though). Do favors for people. Share what you know.

I'd also say what you're doing is really great- and that's- being aware of who your heroes are. Understanding that other people are making dope stuff around you not only broadens your sense of what you can do, but also keeps you from thinking only one way about something. The trick is, not to let anyone else's accomplishments bring you down, but lift you up. They're showing you the finish product of all of their work, not any of the struggle to get there. I have over 1000 private pens on codepen because I'm always messing around and messing up! It's important to understand that most people are only showing you the cream of the crop, so don't get down if you're not there yet when you start out.

Hope that helps answer your question! Good luck!

**Comment by valhead** [comment]
Thanks for posting @Gaoussou1! 

I have pretty similar thoughts to Sarah on this. I think to get better at being a creative developer you always have to have something you're interested in exploring. You need to take the time to learn things that interest you and maybe combine some of your interests in creative ways. It's really difficult to make the time to learn something you're not very interested in.

Making things is the most important step of getting better at any creative skill. Making things is also the hardest part! But not everything you make has to be polished and amazing and shared publicly. Sometimes you just end up making a big giant mess of code things, but you learned something from it, so it was still worth doing. 

I think sharing your knowledge and teaching others helps you get better faster as well. It doesn't necessarily have to be formally teaching or anything like that, but when you can explain what you know to someone else it helps you understand it better. Also, other people have different perspectives and goals and ask questions you might not have thought of. Sharing knowledge really benefits everyone that way.

You've got some good resources listed there. Nature of Code and all of Dan Shiffman's books and tutorials are something I recommend to pretty much everyone who wants to get better at creative coding. It sounds like you're off to a good start! 

And don't be too worried about how long it takes to learn everything you want to learn. If you try to rush it, you won't get to the same results. Learning takes time and practice and that's totally ok.

**Comment by TheOnizu** [comment]
I really like to make thing with javascript and blow minds with my creativity, i guess i just have to keep it that way and try to teach youngster about it.
👍
